docs/app/pages/components/components-sections/tabs/tabs/tabs.component.html

```diff
@@ -13,9 +13,8 @@ <h4>{{ tab.title }}</h4>
             </div>
         </ux-tab>
     </ux-tabset>
-
+    <button class="btn button-primary" (click)="tabset.selectTab(1)">Select Second Tab</button>
```

Comment:
Add the class <code>button-toggle-primary</code> then there is no need for the custom css for the button